packages/disploy/src/structs/ActionRow.ts

```diff
@@ -0,0 +1,23 @@
+import { Base } from "./Base";
+import type { App } from "../client";
+import { ComponentType } from "discord-api-types/v10";
+import type { ActionRowComponent } from "./ActionRowComponent";
+
+export class ActionRow extends Base {
+    private type: ComponentType;
+    private components: Array<ActionRowComponent> = [];
```

Comment:
It's a bit more idiomatic this way. (Same change for the others)
```suggestion
    private components: ActionRowComponent[] = [];
```